package.json

```diff
@@ -271,7 +276,7 @@
         "semantic-release": "^24.2.3",
         "sinon": "^2.3.7",
         "source-map-loader": "^0.2.4",
-        "storybook": "^8.6.14",
+        "storybook": "8.2.4",
```

Comment:
don't merge these storybook dependency changes. this is only for testing right now